src/components/Story.svelte

```diff
@@ -1,42 +1,28 @@
 <script lang="ts">
-  import {LatLng, Map} from 'leaflet';
-  import { onDestroy } from 'svelte';
+  import { onMount } from 'svelte';
   import Slider from '@smui/slider';
-  import { mapStore, layersStore, incomeSlider } from '../stores.js';
+  import { layersStore, incomeSlider } from '../stores.js';
   import Paper, { Title, Subtitle, Content } from '@smui/paper';
+  import config from '../configs/config.json';
+  import { CreateControl } from './controls/ControlCreator.svelte';
 
-  let map: Map|undefined = undefined;
+  let controls = [];
 
-  const unsubscribe = mapStore.subscribe((newMap: Map|null) => {
-    if (newMap != null) {
-      map = newMap;
-    }
-  });
+  onMount(async () => {
+      const controlPromises = config.layers
+        .map((layerConfig: Layer) => layerConfig.controlProperties?.map(l => {
+            return {idLayer: layerConfig.id, nameLayer: layerConfig.name, controlProperties: l}}))

```

Comment:
As discussed, each layer can have multiple controls. I could have just passed the whole layer config to each control, but I didn't want a control to know the configs to all other controls.

As I build out these layers, I am thinking of updating that Layers config to look like
```
export interface Layer {    
  id: string,
  property: LayerProperty,
  controlProperties?: ControlProperty[]
  args?: [a better typed args]    
}  
```
This means the Layer config is really just a mapping, and it would clean up this code to let me pass those properties along to each control. The controls will need those properties to do things like find the correct stores.

Thoughts?

Reply:
What about `const controlPromises = config.layers.map((layerConfig: Layer) => CreateControls(layerConfig)).flat()`

What's in `LayerProperty`?

Reply:
That would work. I will make the change. Thank you for the suggestion.

`LayerProperty` would for now just include the name and the url, but it would give us the flexibility of adding more properties in the future. The problem now, and it is creating some ugly code, is that a control may need to know details about the layer. The name is a great example here. If we pushed that into a `LayerProperty`, then it cleans up for future versions. I can push a fix for that in this PR to clean up this code.